lib/streamaggr/streamaggr_test.go

```diff
@@ -96,21 +96,21 @@ func TestAggregatorsFailure(t *testing.T) {
 	f(`
 - interval: 1m
   staleness_interval: 1foo
-  outputs: ["quantiles"]
```

Comment:
when I change this test case from 1foo to 1m in staleness_interval, the test still passes, because quantiles fails as it's incorrect init of this aggregate function

Reply:
If I understand correctly, this change is not related to PR? If yes, can we rm it and send as a separate PR?

Reply:
removed, test cases are broken though